src/scripts/active-inactive-orgunit/index.js

```diff
@@ -137,28 +136,28 @@ OrganisationUnitActivator.prototype.prepareDataElement = function (dataElement)
     });
 };
 
-/**
+/** 
```

Comment:
Try not to add spaces in lines if nothing changes; introduces noise into the PR without any sense